docs/external-generated/packages/powerpack-azure-cache/documents/overview.md

```diff
@@ -0,0 +1,81 @@
+
+{% callout type="warning" title="Potential Cache Poisoning" %}
+Using your own Azure Storage account to host the remote cache opens you up to the possibility of [cache poisoning](/troubleshooting/unknown-local-cache). To avoid this, use [Nx Replay](/ci/features/remote-cache).
+{% /callout %}
+
+{% callout title="This plugin requires an active Nx Powerpack license" %}
+In order to use `@nx/powerpack-azure-cache`, you need to have an active Powerpack license. If you don't have a license or it has expired, your cache will no longer be shared and each machine will use its local cache.
+{% /callout %}
+
+## Set Up @nx/powerpack-azure-cache
+
+### 1. Install the Package
+
+1. [Activate Powerpack](/nx-enterprise/activate-powerpack) if you haven't already
+2. Install the package
+
+```shell
+nx add @nx/powerpack-azure-cache
+```
+
+### 2. Authenticate with Azure
+
+There are several ways to [authenticate with Azure Storage](https://github.com/Azure/login#login-with-openid-connect-oidc-recommended), but the method recommended by Azure is to use OpenID Connect, like this:
+
+```yaml {% fileName=".github/workflows/ci.yml" %}
+name: CI
+...
+permissions:
+  id-token: write
+  ...
+
+env:
+  NX_DB_CACHE: true
+
+jobs:
+  main:
+    env:
+      NX_POWERPACK_LICENSE: ${{ secrets.NX_POWERPACK_LICENSE }}
+    runs-on: ubuntu-latest
+    steps:
+        ...
+
+      - name: Azure login
+        uses: azure/login@v2
+        with:
+          client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
+        ...
+
+      - run: pnpm exec nx affected -t lint test build
+```
+
+You need to set the `AZURE_CLIENT_ID`, `AZURE_TENANT_ID` and `AZURE_SUBSCRIPTION_ID` secrets as defined in the [Azure documentation](https://github.com/Azure/login#login-with-openid-connect-oidc-recommended).
+
+Note: Any authentication method that [sets up the `DefaultAzureCredentials`](https://learn.microsoft.com/en-us/azure/developer/javascript/sdk/credential-chains#use-defaultazurecredential-for-flexibility) will enable the plugin to work.
+
+### 3. Configure the Nx Cache to Use Azure Storage
+
+Finally, you need to configure your Nx cache in the `nx.json` file. The `container` that you specify needs to already exist - Nx doesn't create it for you.
+
+```jsonc {% fileName="nx.json" %}
+{
+  "azure": {
+    "container": "mycontainer",
+    "accountName": "myaccount"
+  }
+}
+```
+
+| **Property** | **Description**               |
+| ------------ | ----------------------------- |
+| **bucket**   | The name of the bucket to use |
```

Comment:
```suggestion
| **container**   | The name of the container to use |
| **accountName** | The name of blob storage account |
```

sorry, this might need formatting